website/app/GeoKrety/HealthState.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace GeoKrety;
+
+class HealthState {
+    const STATE_OK = 'ok';
+    const STATE_KO = 'ko';
+    const VALID_STATES = [self::STATE_OK, self::STATE_KO];
+
+    public $state = null;
+    public $dependencies = [];
+
+    public function isOk() {
+        return $this->state === self::STATE_OK;
+    }
+
+    public function refreshState() {
+        $this->state = self::STATE_OK;
+        foreach ($this->dependencies as $depValue) {
+            if ($depValue['state'] != self::STATE_OK) {
+                $this->state = self::STATE_KO;
+
+                return;
+            }
+        }
+    }
+
+    public function setDependencyState($dependency, $state, $details = null) {
+        $this->checkValidState($state);
+        $this->dependencies[$dependency]['state'] = $state;
+        if (!is_null($details)) {
+            $this->dependencies[$dependency]['details'] = $details;
+        }
+    }
+
+    public function checkValidState($state) {
+        if (!in_array($state, self::VALID_STATES)) {
+            throw new \Exception("Unhandled state $state");
+        }
+    }
+
+    public function render() {
```

Comment:
Just found in the doc that output is limited to 4096 bytes. Just to keep in mind when we'll implement more tests.

> To help debug failing probes, any output text (UTF-8 encoded) that the command writes on stdout or stderr will be stored in the health status and can be queried with docker inspect. Such output should be kept short (only the first 4096 bytes are stored currently).

https://docs.docker.com/engine/reference/builder/#healthcheck